client/src/Movies/Movie.js

```diff
@@ -18,13 +19,13 @@ const Movie = (props) => {
           console.error(error);
         });
 
-  },[]);
+  },[props.id]);
   
```

Comment:
Nice job passing in the dependency array to this effect hook. Now this useEffect will run anytime the id changes!